dashboard/lib/clients/cache_client.rb

```diff
@@ -0,0 +1,16 @@
+class CacheClient
```

Comment:
The `lti_v1_controller` has some private methods to help with reading and writing to the cache. However, those aren't available outside that controller, so I made a client that can be used by the smaller modular controllers. I chose not to update any existing cache code in the `lti_v1_controller` in an effort to keep this PR smaller, however that could be done via a followup PR.